### ISSUE TITLE
ng command hangs for Node v7.0.0

### ISSUE THREAD

**Issue by jtsom** [angular/angular-cli#2888]
> ## Please provide us with the following information:
### OS?

> Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)

OS X Sierra 10.12.1
Homebrew 1.0.8-133-g84d1661
### Versions.

> Please run `ng --version`. If there's nothing outputted, please run in a Terminal: `node --version` and paste the result here:

v1.0.0-beta.18 installed. Cannot run command, `ng` command hangs
### Repro steps.

> Was this an app that wasn't created using the CLI? What change did you do on your code? etc.

Type `ng` command. Nothing happens. Status bar in Terminal window shows: ng --version(node)

Have to Control-C to exit
### The log given by the failure.

None.
### Mention any other details that might be useful.

> ---
> 
> Thanks! We'll be in touch soon.

**Comment by jtsom** [comment]
Ok, digging around, looks like it's a problem with watchman. I uninstalled that - after going back to Node 6.9.1 and `ng` still wasn't working, saw that watchman was trying to run. Uninstalling watchman  seems to have worked.

**Comment by mixomat** [comment]
@jtsom thx, had the same issue 👍

**Comment by jtsom** [comment]
Strange thing is, on another system, having Watchman installed is not causing any problems.

I can run Node 7 just fine with angular-cli without watchman on this system. Add watchman, and it hangs. (For what it's worth, the Ember cli command does the same - not surprising as they're similar)

**Comment by jtsom** [comment]
More testing. I tried installing watchman again, and ng was still hanging.

This time I uninstalled watchman (`brew uninstall watchman`) and also removed the launchctl item (`launchctl remove com.github.facebook.watchman`) to make sure it was gone.

`ng` worked fine. So I reinstalled watchman and started up an angular 2 app - and it seems to be ok now. The launchctl item reinstalled itself - I can only assume something got corrupted?

YMMV
